highfidelity/hifi
# Fix marketplacesInject.js for local Metaverse

This didn't work before because `metaverse.highfidelity.com/marketplace` automatically redirects to `highfidelity.com/marketplace`. This fix isn't very pretty, but it works. The benefits of this fix are really only for developers who use local metaverses during dev/testing, but we should verify that this change doesn't break anything.

**Test Plan:**
*PREREQ:* Don't modify your `HIFI_METAVERSE_URL` environment variable. Don't log in.
1. Open the MARKET app. Verify that you see a green banner towards the top of the app that says "LOG IN to purchase items from the Marketplace."
2. Click on a costed item to go to its individual item page. Verify that you see a green banner towards the top of the app that says "LOG IN to purchase items from the Marketplace."
3. Click on the "PURCHASE" button. Verify that you see the "HIFI COMMERCE - LOGIN" screen.
4. Quit Interface. Modify your `HIFI_METAVERSE_URL` environment variable such that the value is `https://staging.highfidelity.com`. Launch Interface and repeat steps 1-3.
